charts/shoot-core/charts/calico/templates/psp/calico-kube-controllers-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: garden.sapcloud.io:psp:kube-system:calico-kube-controllers
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups:
+  - policy
+  - extensions
```

Comment:
```
The node controller automatically cleans up configuration for nodes that no longer exist. The controller must have read access to the Kubernetes API to monitor Node events.
```

Ref https://docs.projectcalico.org/v3.7/reference/kube-controllers/configuration

Reply:
You could ignore that.